pennylane/ops/qutrit/non_parametric_ops.py

```diff
@@ -202,3 +203,206 @@ def adjoint(self):
         op = TClock(wires=self.wires)
         op.inverse = not self.inverse
         return op
+
+
+class TAdd(Operation):
+    r"""TAdd(wires)
+    The 2-qutrit controlled add gate
+
+    The construction of this operator is based on definition 7 from
+    `Yeh et al. (2022) <https://arxiv.org/abs/2204.00552>`_.
+    It performs the controlled :class:`~.TShift` operation, and sends
+    :math:`\hbox{TAdd} \vert i \rangle \vert j \rangle = \vert i \rangle \vert i + j \rangle`,
+    where addition is taken modulo 3. The matrix representation is
+
+    .. math:: TAdd = \begin{bmatrix}
+                        1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\
+                        0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 & 0 \\
+                        0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 & 0 \\
+                        0 & 0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 \\
+                        0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 & 0 \\
+                        0 & 0 & 0 & 0 & 1 & 0 & 0 & 0 & 0 \\
+                        0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 \\
+                        0 & 0 & 0 & 0 & 0 & 0 & 0 & 0 & 1 \\
+                        0 & 0 & 0 & 0 & 0 & 0 & 1 & 0 & 0
+                    \end{bmatrix}
+
+    .. note:: The first wire provided corresponds to the **control qutrit**.
+
+    **Details:**
+
+    * Number of wires: 2
+    * Number of parameters: 0
+
+    Args:
+        wires (Sequence[int]): the wires the operation acts on
+    """
+    num_wires = 2
+    """int: Number of wires that the operator acts on."""
+
+    num_params = 0
+    """int: Number of trainable parameters that the operator depends on."""
+
+    @staticmethod
+    def compute_matrix():
+        r"""Representation of the operator as a canonical matrix in the computational basis (static method).
+
+        The canonical matrix is the textbook matrix representation that does not consider wires.
+        Implicitly, this assumes that the wires of the operator correspond to the global wire order.
```

Comment:
I think this line might also add to the confusion when it mentions a global wire order

Reply:
I do agree with you. However, as @glassnotes mentioned in another comment, this comment is used for the `compute_matrix` method of every other operation as well, which is why I'm hesitant to change it at the moment.

Reply:
So if I understand correctly this is the same docstring as those for most of the other operations in PL? or in the qutrit project?